src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.cs

```diff
@@ -16,6 +16,26 @@
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
+    internal struct Alias
+    {
+        internal Alias(PseudoVariableKind kind, string name, string fullName, string type, DkmClrCustomTypeInfo customTypeInfo)
+        {
+            this.Kind = kind;
+            this.Name = name;
+            this.FullName = fullName;
+            this.Type = type;
+            this.CustomTypeInfo = customTypeInfo;
+        }
+
+        internal readonly PseudoVariableKind Kind;
+        internal readonly string Name;
+        internal readonly string FullName;
+        internal readonly string Type;
+        internal readonly DkmClrCustomTypeInfo CustomTypeInfo;
```

Comment:
I don't think we can instantiate this type in our unit tests, so it might be preferable to either unwrap it or pass around a DynamicFlagsCustomTypeInfo.